samples/apps/copilot-chat-app/webapi/appsettings.json

```diff
@@ -171,6 +171,35 @@
     }
   },
 
+  //
+  // ChatSkill prompts are used to generate responses to user messages.
+  // - CompletionTokenLimit is the token limit of the chat model, see https://platform.openai.com/docs/models/overview
+  // - ResponseTokenLimit is the token count left for the model to generate text after the prompt.
+  //
+  "Prompts": {
+    "CompletionTokenLimit": 4096,
+    "ResponseTokenLimit": 1024,
+
+    "SystemDescription": "This is a chat between an intelligent AI bot named Copilot and {{$audience}}. SK stands for Semantic Kernel, the AI platform used to build the bot. The AI was trained on data through 2021 and is not aware of events that have occurred since then. It also has no ability to access data on the Internet, so it should not claim that it can or say that it will go and look things up. Try to be concise with your answers, though it is not required. Knowledge cutoff: {{$knowledgeCutoff}} / Current date: {{TimeSkill.Now}}.",
```

Comment:
Following statement conflicts with line 186 :"The AI was trained on data through 2021" 
![image](https://github.com/microsoft/semantic-kernel/assets/39630244/de5436ca-d9b1-4d0b-bd1f-3c4eee1f43e6)

Reply:
"through 2021" cuts off on Jan 1, 2022. I think this ends up reinforcing the same timeline.